File: CS/CSPDashboardExample/Controllers/DefaultDashboardController.cs
using CSPDashboardExample.Models;
using DevExpress.DashboardAspNetCore;
using DevExpress.DashboardWeb;
using Microsoft.AspNetCore.DataProtection;
using Microsoft.AspNetCore.Mvc;

namespace CSPDashboardExample.Controllers {
    public class DefaultDashboardController : DashboardController {
        public DefaultDashboardController(DashboardConfigurator configurator, IDataProtectionProvider? dataProtectionProvider = null)
            : base(configurator, dataProtectionProvider) {
        }

        public IActionResult Index()
        {
            return View(new DashboardModel());
        }
    }
}

File: VB/CSPDashboardExample/Controllers/DefaultDashboardController.vb
Imports CSPDashboardExample.Models
Imports DevExpress.DashboardAspNetCore
Imports DevExpress.DashboardWeb
Imports Microsoft.AspNetCore.DataProtection
Imports Microsoft.AspNetCore.Mvc

Namespace CSPDashboardExample.Controllers

    Public Class DefaultDashboardController
        Inherits DashboardController

        Public Sub New(ByVal configurator As DashboardConfigurator, ByVal Optional dataProtectionProvider As IDataProtectionProvider? = Nothing)
            MyBase.New(configurator, dataProtectionProvider)
        End Sub

        Public Function Index() As IActionResult
            Return View(New DashboardModel())
        End Function
    End Class
End Namespace
